rainbow-me/rainbow
# [RNBW-1828] Move Uniswap Pools to Dispersion

[Dispersion PR](https://github.com/rainbow-me/dispersion/pull/2)

Fixes RNBW-1828

**QA instructions**:
This feature changed the way we retrieve data for our pool UI. This affects pools shown in the user's wallet, on the discover page, and the pools featured on the bottom of a token's chart expanded state. We are caching the data every few minutes, so this pool data may not be an exact match with production but we should check that the values are quite close to each other if not identical.
The purpose of this feature was to remove unnecessary requests in the app and speed up retrieval of pool data. So, speed should be compared against prod. Slower rendering of pool UI would be considered a blocker and may indicate an issue.
LMK if you have any questions @ibrahimtaveras00
